tests/System.Text.Primitives.Tests/Parsing/PrimitiveParserSBytePerfTests.cs

```diff
@@ -11,6 +11,30 @@ public partial class PrimitiveParserSBytePerfTests
     {
         private const int InnerCount = 100000;
 
+        private static readonly string[] s_SByteTextArray = new string[20]
```

Comment:
This likely doesn't need to be 20 strings long. You have some duplicate strings being tested, like "2", etc. I would remove (or change) any duplicates.